src/AddressBook/ABAddressBook.cs

```diff
@@ -200,15 +200,15 @@ public void RequestAccess (Action<bool, NSError?> onCompleted)
 			}
 		}
 
-		internal delegate void InnerCompleted (IntPtr block, bool success, IntPtr error);
+		internal delegate void InnerCompleted (IntPtr block, byte success, IntPtr error);
 		static readonly InnerCompleted static_completionHandler = TrampolineCompletionHandler;
 		[MonoPInvokeCallback (typeof (InnerCompleted))]
```

Comment:
should this (and the other) have `[UnmanagedCallersOnly]` or are you planning to do that later?

Reply:
That's coming later